app/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE resources [
+    <!ENTITY appname "Slideshow">
+    <!ENTITY author "Michael Standen">
+    ]>
+<resources>
+    <string name="app_name"></string>
```

Comment:
@JekRock I'm not sure, but this guy seems like he does [stackoverflow](http://stackoverflow.com/a/7947337)